consent/handler.go

```diff
@@ -231,15 +232,27 @@ type revokeOAuth2LoginSessions struct {
 	// in: query
 	// required: true
```

Comment:
Not sure how to make the API accept either `subject` or `sid` without introducing a breaking change to this schema.

Reply:
Hm good question, I think in that case it's ok to break the schema here and we'll release it as a minor patch

Reply:
As it is implemented now there is no breaking change, because we remove required, and add a new parameter. So old clients will continue to work.